sourcemash/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 def create_app(env=None):
 	app = Flask(__name__, instance_relative_config=True,
-				template_folder='frontend/templates',
+				template_folder='frontend/static/templates',
```

Comment:
Forgot I did that. I might be needed to use the JST object, but I'll try moving it back out.